charts/castai-hosted-model/child-charts/vllm/templates/secret.yaml

```diff
@@ -0,0 +1,10 @@
+{{- if .Values.hfToken }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ include "vllm.fullname" . }}-{{ now | date "02012006150405" }}
```

Comment:
Do we need this date? I assume the `vllm.fullname` will be unique anyway.

Reply:
This date is to handle the case of same 2 models being deployed in 2 different operations. However, I will update the fullname to actually be the service name since it must be unique on our side. Just like we have for Ollama.